packages/ui/src/components/Tag/tag.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { CSSProperties } from 'react'
+import {
+    Tag as DefaultTag,
+} from 'antd'
+import { colors } from '@open-condo/ui/src/colors'
+
+const TAG_CLASS_PREFIX = 'condo-tag'
+
+export type TagProps = React.HTMLAttributes<HTMLSpanElement> & {
+    children: string
+    textColor?: CSSProperties['color']
+    bgColor?: CSSProperties['backgroundColor']
+}
+
+const Tag = React.forwardRef<HTMLSpanElement, TagProps>((props, ref) => {
+    const {
+        children,
+        textColor = colors.gray['7'],
```

Comment:
literal constants aren't good for understanding what's going on and why. better to name them